packages/react-components/react-calendar-compat/library/src/components/CalendarDayGrid/CalendarGridDayCell.tsx

```diff
@@ -237,6 +240,7 @@ export const CalendarGridDayCell: React.FunctionComponent<CalendarGridDayCellPro
       tabIndex={isNavigatedDate || isFocusable ? 0 : undefined}
       aria-current={day.isToday ? 'date' : undefined}
       aria-selected={day.isInBounds ? day.isSelected : undefined}
+      {...focusAttributes}
```

Comment:
@mshoho  can you please check `isDefault` would be needed here. The example seems pretty straightforward that this should not be needed